odoo80/Dockerfile

```diff
@@ -1,51 +1,5 @@
-FROM vauxoo/docker-ubuntu-base
+FROM vauxoo/docker-ubuntu-base:latest
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
 
-RUN wget -qO- https://deb.nodesource.com/setup | sudo bash - \
```

Comment:
I [changed this](https://github.com/Vauxoo/docker-odoo-image/pull/104/commits/7b4fdbcfb35a9578b888154c04acadfbbd8bb007#diff-47daa48a95db7533add915692de62084R48) script for the dpkg node upstream repo, which is more up to date than this.